2.75

ATSchoolENikonov opened the pull request:
Выдает инфу

methshest committed:
2.75
Выдает инфу

Separador commented on Home_2.75/Task_1.java:
```diff
@@ -0,0 +1,29 @@
+                    str[i] = str[i].replace(str[i], "");
+                }
+            }
+            if (str[x].equals("")) {
+                System.out.print(str[x]);
+            } else {
```
Не совсем понял для чего печатать пустые слова. Лучше будет, если сделать проверку, что слово непустое и тогда его печатать.

Separador reviewed (changes requested):
Хорошо, поправь момент с условием

methshest committed:
2.75
Что-то вышло, удаления фрукта не использвоал, так как у Листа уже есть метод удаления)
